app/tests/test_incident.py

```diff
@@ -0,0 +1,24 @@
+from app.tests.test_base import BaseTest
+from .test_base import corruptioncasemodel
+from app import app
+import json
+
+class TestOneRedFlag(BaseTest):
+    
+
+    # def test_get_one_redflag(self):
+    #     response = self.client.get('/api/v1/redflagrecords/1')
+    #     data =  json.loads(response.data)
+    #     self.assertEqual(data["status"],404)
```

Comment:
@Wabuluka 
the response also returns an object .status_code.
You can try asserting  your test like this

`response.status_code`